src/plugins/index_pattern_management/public/components/index_pattern_table/index_pattern_table.tsx

```diff
@@ -140,6 +111,35 @@ export const IndexPatternTable = ({ canSave, history }: Props) => {
 
   chrome.docTitle.change(title);
 
+  const columns = [
+    {
+      field: 'title',
+      name: 'Pattern',
+      render: (
+        name: string,
+        index: {
+          id: string;
+          tags?: Array<{
+            key: string;
+            name: string;
+          }>;
+        }
+      ) => (
+        <EuiButtonEmpty size="xs" {...reactRouterNavigate(history, `patterns/${index.id}`)}>
+          {name}
+          {index.tags &&
```

Comment:
Nit: I think this needs a bit more space between name and tags
Old:
<img width="338" alt="Bildschirmfoto 2020-06-02 um 12 17 22" src="https://user-images.githubusercontent.com/463851/83509015-0eb0b700-a4cb-11ea-8d76-328cca1a0a2d.png">
New: 
<img width="338" alt="Bildschirmfoto 2020-06-02 um 12 04 50" src="https://user-images.githubusercontent.com/463851/83509038-183a1f00-a4cb-11ea-8b2a-db3252a8b9fa.png">